tools/dependencies/known-dependencies.txt

```diff
@@ -34,4 +34,7 @@ j2objc-annotations-1.1.jar
 jsr305-1.3.9.jar
 jsr305-3.0.0.jar
 jsr305-3.0.2.jar
-listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar
\ No newline at end of file
+listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar
```

Comment:
When you add a new jar package, you first need to process its license,  you can find other jars license in `seatunnel-dist/release-docs/LICENSE`. When you process its license complete, you can update `tools/dependencies/known-dependencies.txt` to fix the dependency check.